pyaedt/hfss.py

```diff
@@ -479,37 +484,17 @@ def create_linear_count_sweep(self, setupname, unit, freqstart, freqstop, num_of
         <class 'pyaedt.modules.SetupTemplates.SweepHFSS'>
 
         """
-
-        if sweepname is None:
-            sweepname = generate_unique_name("Sweep")
-
-        if setupname not in self.setup_names:
+        if sweep_type not in ["Discrete", "Interpolating", "Fast"]:
+            self.add_error_message("Error in Sweep Type. It has to be one of Discrete, Interpolating or Fast")
```

Comment:
```suggestion
            self.add_error_message("Error in Sweep Type. It has to either 'Discrete', 'Interpolating', or 'Fast'")
```

Reply:
Also, shouldn't we be raising a `ValueError` here?

Reply:
probably yes